cmd/kubernetes-history-inspector/main.go

```diff
@@ -41,63 +42,64 @@ import (
 	"github.com/GoogleCloudPlatform/khi/pkg/source/gcp/api/accesstoken"
 	"github.com/GoogleCloudPlatform/khi/pkg/source/gcp/api/quotaproject"
 
-	"cloud.google.com/go/profiler"
+	"log/slog"
 )
 
-func displayStartMessage(host string, port int) {
-	var (
-		bold  = "\033[1m"
-		green = "\033[32m"
-		cyan  = "\033[36m"
-		reset = "\033[0m"
-	)
-	if parameters.Debug.NoColor != nil && *parameters.Debug.NoColor {
-		bold = ""
-		green = ""
-		cyan = ""
-		reset = ""
-	}
-	hostInHintText := host
-	if host == "0.0.0.0" || host == "127.0.0.1" {
-		hostInHintText = "localhost"
-	}
-	fmt.Printf(`%[1]s%[2]s%[3]s Starting KHI server with listening %[4]s:%[5]d%[1]s`, reset, bold, green, host, port)
-	if hostInHintText == "localhost" {
-		fmt.Printf(`
-%[4]s%[2]sFor Cloud Shell users:
-	Click this address >> %[3]shttp://%[5]s:%[6]d%[1]s%[2]s%[4]s << Click this address
-
-%[1]s%[4]s(For users of the other environments: Access %[3]shttp://%[5]s:%[6]d%[1]s%[4]s with your browser. Consider SSH port-forwarding when you run KHI over SSH.)
-%[1]s`, reset, bold, green, cyan, hostInHintText, port)
-	}
-}
-
-var taskSetRegistrer []inspection.PrepareInspectionServerFunc = make([]inspection.PrepareInspectionServerFunc, 0)
+var taskSetRegistrer []inspection.PrepareInspectionServerFunc
 
 func init() {
+	// Add the different parameter stores.
 	parameters.AddStore(parameters.Help)
 	parameters.AddStore(parameters.Common)
 	parameters.AddStore(parameters.Server)
 	parameters.AddStore(parameters.Job)
 	parameters.AddStore(parameters.Auth)
 	parameters.AddStore(parameters.Debug)
 
-	taskSetRegistrer = append(taskSetRegistrer, common.PrepareInspectionServer)
-	taskSetRegistrer = append(taskSetRegistrer, gcp.PrepareInspectionServer)
+	// Register inspection server preparation functions.
+	taskSetRegistrer = []inspection.PrepareInspectionServerFunc{
+		common.PrepareInspectionServer,
+		gcp.PrepareInspectionServer,
+	}
```

Comment:
We can't directly assign the value to the variable because of the reason that I commented few lines before. But thank you for adding comment to make it clear.

```suggestion
	// Register inspection server preparation functions.
	taskSetRegistrer = append(taskSetRegistrer, common.PrepareInspectionServer)
	taskSetRegistrer = append(taskSetRegistrer, gcp.PrepareInspectionServer)
```